src/OrchardCore.Modules/OrchardCore.Lists/Services/ContainerService.cs

```diff
@@ -339,9 +355,43 @@ public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string cont
             }
         }
 
-        private static Expression<Func<ContentItemIndex, bool>> CreateOrderedContentIndexFilter(bool publishedOnly)
+        private IQuery<ContentItem> ContainedItemOptionsFilter(ContainedItemOptions containedItemOptions, IQuery<ContentItem> containedItems)
+        {
+            if (!string.IsNullOrEmpty(containedItemOptions.DisplayText))
+            {
+                containedItems.With<ContentItemIndex>().Where(i => i.DisplayText.Contains(containedItemOptions.DisplayText));
+            }
+
+            if (containedItemOptions.Status != ContentsStatus.None)
```

Comment:
Get rid of this None.

If it was ever used it would result in the entire history of the list documents being returned.

Something noone should see ;)